packages/manager/modules/pci/src/projects/project/instances/add/add.less

```diff
@@ -87,4 +87,20 @@ ovh-manager-pci-instances-add {
       vertical-align: middle;
     }
   }
+
+  .oui-field-font-size {
+    font-size: 0.875rem !important;
+  }
+
+  .existing-network {
+    oui-field > div {
+      margin-bottom: 1rem !important;
+    }
+  }
+
+  .oui-message-localzone {
+    .oui-message {
+      padding: 1rem 1.5rem 0.5rem 3rem !important;
```

Comment:
**nitpick:** Dont use custom values.

Reply:
The design team requested these style changes